app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java

```diff
@@ -41,17 +44,21 @@
     private Button btn2;
     private TextView exchange;
     private DocumentReference bookDocument;
+    private boolean goodScan;
     private RequestDetailViewFragment requestDetailViewFragment;
     private Resources resources;
     private ConstraintLayout dropdownContainer;
 
+    public static final int REQUEST_GIVE_SCAN = 5;
+    public static final int REQUEST_RECIEVE_SCAN = 6;
```

Comment:
Pls correct typo to REQUEST_RECEIVE_SCAN. Select this, press and hold ALT J to select all occurrences and change at once.

Reply:
ngl that's kind of embarrassing for me but thanks for catching this